app/src/main/res/xml/preferences_user_interface.xml

```diff
@@ -87,5 +87,6 @@
                 android:key="prefSwipe"
                 android:summary="@string/swipeactions_summary"
                 android:title="@string/swipeactions_label"/>
+
```

Comment:
Please revert the added line to keep the PR diff more clean

Reply:
Sure, the empty line removed.